source/command-line-options.txt

```diff
@@ -33,18 +35,95 @@ Use the following syntax to start |compass| and set a configuration option:
 Options
 -------
 
-You can set the following configuration options using the command line:
+You can set these options when using the command line. 
+
+Command Line Only
+~~~~~~~~~~~~~~~~~
+
+These options can only be set on the command line. These options
+cannot be set in the :guilabel:`Settings` panel.
+
+.. option:: --exportConnections
+    
+   :ref:`Export <compass-export-connections-cli>` favorite connections.
+
+.. option:: --file <file>
+  
+   Specify a file containing a :ref:`list of connections 
+   <compass-export-connections-cli>`. These connections are 
+   automatically available once |compass| starts.
+
+.. option:: --help
+   
+   Show |compass| options.
+
+.. option:: --importConnections
+
+   :ref:`Import <compass-import-connections-cli>` favorite connections.
+
+.. option:: --passphrase
+ 
+   Specify a passphrase to encrypt the :ref:`exported favorite 
+   connections <compass-export-connections-cli-encrypted>` output file.
+
+   Specify a passphrase to decrypt the :ref:`imported favorite
+   connections <compass-import-connections-cli-encrypted>` file. 
+
+.. option:: --version
+
+   Show |compass| version.
+
+Command Line or Settings Panel
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+These options can be set on the command line **or** in the 
+:guilabel:`Settings` panel. Setting an option on the command line 
+overrides the value in the :guilabel:`Settings` panel.        
+
+.. option:: --autoUpdates
+  
+   Enable :ref:`automatic updates <enable-automatic-updates>`. Use 
+   ``--no-autoUpdates`` to disable automatic updates. 
+
+.. option:: --enableFeedbackPanel
+  
+   Enable the feedback panel. Use ``--no-enableFeedbackPanel`` to 
+   disable automatic updates. 
+
+.. option:: --enableMaps
+ 
+   Allow |compass| to make requests to a 3rd party mapping service. Use
+   ``--no-enableMaps`` to disable mapping requests.
+
+.. option:: --lgDarkmode
+
+   Use leafygreen dark mode instead of darkreader.
```

Comment:
[question]

Will users know what "leafygreen" dark mode is / means? Is there any difference between the two options we might want to highlight here?

Reply:
Thank you for bringing this up. I don't have any other info. I'm going to remove this option.

Reply:
Sounds good, thanks!